Add -tplR option

hlruh opened the pull request:
Option allows to read in and broaden a 1d template spectrum. The 1d template spectrum can be generated from a serval template from observations with another instrument using tpl1d.py.

hlruh committed:
Add -tplR option

Option allows to read in and broaden a 1d template spectrum. The 1d template spectrum can be generated from a serval template from observations with another instrument using tpl1d.py.

hlruh committed:
Add ipbroad

Add a function to broaden a template spectrum by a gaussian instrumental profile. Additional parameter R for template class Tpl.

mzechmeister commented on src/serval.py:
```diff
@@ -1327,6 +1327,52 @@ def serval():
                   if omin<head['HIERARCH SERVAL COADD COMIN']: pause('omin to small')
                   if omax>head['HIERARCH SERVAL COADD COMAX']: pause('omax to large')
                TPLrv = head['HIERARCH SERVAL TARG RV']
+            elif tpl.endswith('.tpl%s.fits'%fibsuf) or os.path.isdir(tpl):
```
There seems to be some duplication. L1318+ seems very similar.

hlruh replied:
Deleted in new commit.

hlruh committed:
Modify -tplR option

tplR can be set to R_inst (resolving power instrument) or R_inst, R_tpl (resolving power template spectrum). Delete duplicated code  L1318+.

mzechmeister committed:
fix: R computation. chore: fwhm computation simplified. refact: tplR args handling.

mzechmeister committed:
chore: New template header keyword R_inst.
